bedrock/settings/static_media.py

```diff
@@ -1006,12 +1006,6 @@
         ),
         'output_filename': 'css/contribute-signup-bundle.js',
     },
-    'existing': {
-        'source_filenames': (
-            'js/newsletter/existing.js',
```

Comment:
Note: I'm removing this file as all it seems to be responsible for is GA tracking, and has been broken since before we migrated to GTM. I'm not sure how to best re-implement it for GTM without sending crap loads of events, so it would likely need re-working anyhow. No body seems to have missed the data in well over a year.